hudi-common/src/main/java/org/apache/hudi/internal/schema/convert/AvroInternalSchemaConverter.java

```diff
@@ -68,6 +68,17 @@ public static Schema convert(InternalSchema internalSchema, String name) {
     return buildAvroSchemaFromInternalSchema(internalSchema, name);
   }
 
+  /**
+   * Convert avro Schema to avro Schema.
+   *
+   * @param internalSchema internal schema.
+   * @param name the record name.
+   * @return an avro Schema.
+   */
+  public static Schema fixNullOrdering(Schema schema) {
```

Comment:
unit test please.. All these methods are being used in HoodieSparkSqlWrite.deduceWriteSchema. Important to get it right.

Reply:
Why do we need to do this conversion?

Reply:
yes, an example in the comment as to why we need this conversion will be helpful for other readers

Reply:
addressed why we need this in comment. Still need to add test case